src/xgov-dapp/src/features/vote/VoteResults.tsx

```diff
@@ -34,6 +35,28 @@ export const VoteResults = ({
   const [isDownloadingCsv, setIsDownloadingCsv] = useState(false)
   const [error, setError] = useState<string | null>(null)
 
+  const oIdsToCounts = useMemo(() => {
+    if (votingRoundResults === undefined) {
+      return undefined
+    }
+    return Object.fromEntries(votingRoundResults?.map((result) => [result.optionId, result.count]))
+  }, [votingRoundResults])
+
+  const passedPercentage = (q: Question) => {
+    if (!oIdsToCounts || q.options.length === 0 || !q.metadata || !q.metadata.threshold) {
+      return 0
+    }
+    const votesTally = oIdsToCounts[q.options[0].id] ? oIdsToCounts[q.options[0].id] : 0
+    return Math.min(100, (votesTally / q.metadata.threshold) * 100)
+  }
+
+  const isReserveList = (question: Question) => {
+    const percentage = passedPercentage(question)
+    return percentage < 100 && percentage >= 60 // TODO: make this configurable
+  }
+
+  // clone the voting round metadata and adjust the threshold to be out of total votes instead of total voting power
+  // we clone the metadata so that we don't mutate the original metadata
   const votingRoundMetadataClone = useMemo<VotingRoundMetadata | undefined>(() => {
     if (
```

Comment:
Return the original data without modifications if feature flag `isReserveListEnabled` is false